tensorflow_datasets/text/yelp_polarity_test.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for yelp dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.text import yelp_polarity
+
+
+class YelpPolarityReviewsTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = yelp_polarity.YelpPolarityReviews
+  SPLITS = {
+    "train": 5,
+    "test": 4,
+  }
```

Comment:
Also add your `fake_samples` to the appropriate folder.

Reply:
Yes, seems that fake_examples are missing from the PR.